src/internet_identity/src/anchor_management/registration.rs

```diff
@@ -118,11 +120,30 @@ fn create_captcha<T: RngCore>(rng: T) -> (Base64, String) {
     return (resp, captcha.chars_as_string());
 }
 
+lazy_static! {
+    /// Map of problematic characters that are easily mixed up by humans to the "normalized" replacement.
+    /// I.e. the captcha will never contain any of the characters in the key set and any input provided
+    /// will be mapped to the matching value.
+    /// Note: the captcha library already excludes the characters o, O and 0.
```

Comment:
No, I suspect it is because for those characters it is less controversial that they are ambiguous. Contributing more to the library could also help us make use of more filters.

Happy to discuss offline.